app/views/categories/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="categories--container">
+  <article class='form--header'>
+    <h2>Categories</h2>
+    <%= link_to 'LOG OUT', destroy_user_session_path, data: {turbo_method: :delete} %>
+  </article>
+  <article class="catergories--list--container">
+    <ul class="categories--list">
+      <% @categories.each do |category| %>
+        <li class="categories--item">
+        <div class="categories--item--info">
+          <% if category.icon.attached? %>
+          <%= image_tag category.icon %>
+          <% else %>
+            <%= image_tag "https://media.designrush.com/inspiration_images/134933/conversions/_1511456189_555_McDonald's-preview.jpg" %>
+          <% end %>
+          <%= link_to category.name, category_expenses_path(category) %>
```

Comment:
- [ ] Great job working so far but I think it would be great if you align the categories text in the middle so kindly as mentioned by the previous reviewer As required for the project your app to follow the [the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) from Behance, kindly ensure that you update your current design by using the typography(font, size, weight,...), text-transformation, Layout(alignment and position of elements), colors, etc.✅💯🚀✨

- Yours
![image](https://github.com/Akorede3133/expense-insight/assets/58553711/e9106269-7f55-42f8-871c-4baba2c00d5b)

- Given
![image](https://github.com/Akorede3133/expense-insight/assets/58553711/51e5e0e1-08ec-4f4b-9f34-24b19e843202)